packages/spanner/system-test/spanner.js

```diff
@@ -108,6 +108,22 @@ var spanner = new Spanner(env);
       }, execAfterOperationComplete(done));
     });
 
+    describe('uneven rows', function() {
+      it('should allow inserting uneven rows', function(done) {
+        table.insert([
+          {
+            Key: generateName('id'),
+            BoolValue: true
+          },
+          {
+            Key: generateName('id'),
+            BoolValue: true,
+            IntValue: 10
+          }
+        ], done);
```

Comment:
Does that mean simply not accepting this change, and keeping our code the way it is currently?